<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dash_tester</class>
 <widget class="QMainWindow" name="dash_tester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Warnings</string>
    </property>
    <widget class="QCheckBox" name="chkBattery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Low</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkTemp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>High Temperature</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkDamage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Damage</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkService">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Service</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Driving Mode</string>
    </property>
    <widget class="QRadioButton" name="optEco">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Eco Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optNormal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optSport">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sport Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optOffRoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Off-Road Mode</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Vehicle Data</string>
    </property>
    <widget class="QSlider" name="hsbSpeed">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbPower">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>State of Charge:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="hsbSoc">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Range:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbRange">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tripmeter: </string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbTrip">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblSpeed">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblPower">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblSoc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblRange">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblTrip">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Battery Data</string>
    </property>
    <widget class="QSlider" name="hsbBVoltage">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Voltage:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbBTemp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Temp:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cell Voltages:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 1:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 2:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 3:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 4:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 8:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 5:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 6:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell 7:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsbC7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblC1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblC8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblBVoltage">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblBTemp">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Charging Door</string>
    </property>
    <widget class="QRadioButton" name="optOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optClose">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Charging Mode</string>
    </property>
    <widget class="QRadioButton" name="optFast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fast Charge</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optSlow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Slow Charge</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Listening</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
